src/encoded/static/components/matrix_sescc_stem_cell.js

```diff
@@ -324,15 +324,13 @@ const MatrixHeader = ({ context }) => {
 
     return (
         <div className="matrix-header">
-            <div className="matrix-header__title matrix-header__title-sescc-spacing">
-                <div className="matrix-tag">
+            <div className="matrix-header__title">
+                <div className="matrix-title-badge">
                     <h1>{context.title}</h1>
+                    <MatrixBadges context={context} />
```

Comment:
Why no type property?

Reply:
SESCC displays badges based on internal_tags. We need `type` only for matrices not defined by internal_tags.